src/services/generator.service.ts

```diff
@@ -0,0 +1,28 @@
+import AsyncAPIGenerator from '@asyncapi/generator';
+import { AsyncAPIDocument } from '@asyncapi/parser';
+
+import { prepareParserConfig } from "../utils/parser";
+
+/**
+ * Service providing `@asyncapi/generate` functionality.
+ */
+export class GeneratorService {
```

Comment:
```suggestion
export class GenerateService {
```

And any other related thing should be renamed.